examples/_extract_wind_speed_at_points.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2021 NREL
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# See https://floris.readthedocs.io for documentation
+
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import floris.tools.visualization as wakeviz
+from floris.tools import FlorisInterface
+from floris.tools.visualization import (
+    calculate_horizontal_plane_with_turbines,
+    visualize_cut_plane,
+)
+
+
+"""
+Test the use of add points
```

Comment:
It would be helpful to describe what this example is intending to demonstrate. For example, it's noted below that this simulates a met mast between turbines. That would be good to explain at the top maybe with a little diagram of the layout of the turbines vs met mast or at least mention it. Initially, I didn't know what I was looking at in the plot and then went in to read the code.

Reply:
Excellent, thanks for addressing this.